### PR TITLE
FRONTEND WITH FRAMEWORK: REACT 2.4 => Import, Export, Modules doc

### LINKED ISSUE #1757: FRONTEND WITH FRAMEWORK: REACT 2.4 => Import, Export, Modules
Contributors are required to prepare documentation to include information about the Import, Export, modules and anything related to the Import, Export, Module.

Also make sure -
- Avoid copy/paste, We will be checking.
- **Changes should be made inside the Frontend_Web_Development_React_Angular_Vue/ dir inside Frontend_Web_Development_React_Angular_Vue branch.**
- Task will be assigned on first come first serve.
- **Files need to be added into the Week2 folder only.**
- Check out <a href="https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md">Contribution Guidelines</a> and <a href="https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md" >Code Of Conduct</a>.

### PR DESCRIPTION
**Pull request by YASHMAROO**
<hr>

## Description 📜

Added documentation for React imports, exports and modules with proper information and examples for better understanding.
Fixes #1757

<hr>

## Type of change 📝

- [x] Doumentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev #React/Angular/Vue

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

### CODE REVIEW

**Review by heet-kumar — CHANGES_REQUESTED** [review]
This images are not loaded so you have updated in your forked repo from their you create link of the image and replace the image source. from GitHub you can directly can create image source. If you are unable to resolve then mention me in the discord so on call I will explain you.

File: Frontend_Web_Development_React_Angular_Vue/React/Week 2/2.4_Import,Export,Modules.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+The plain, simple English dictionary meaning of the words **import** and **export** are roughly defined as
+> to bring in/out across the border
+
+Similarly, these words explain a kind of **border-crossing** in the land of JavaScript.
+
+Whenever you set up a basic react project you are provided with a _boilerplate_ for your project and henceforth you start building your project on it.
+
+<div style="display: flex; justify-content: center;">
+    <img src="./assets/ReactSetup.png" style="height:   300px; width: auto; ">
```
Comment (heet-kumar) [inline Frontend_Web_Development_React_Angular_Vue/React/Week 2/2.4_Import,Export,Modules.md:21]: same here also for image

File: Frontend_Web_Development_React_Angular_Vue/React/Week 2/2.4_Import,Export,Modules.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+```javascript
+export { NAMED_VAL, ... }
+```
+
+**Default Values vs Named Values** - When multiple components are being imported/exported we use Named Values (those in curly braces **{}**)and for a single module exported as a whole we use default values.
+
+**Default Import**
+<br>
+<div style="display: flex; justify-content: center;">
+    <img src="./assets/Default_Import.png" style="width: auto; ">
```
Comment (heet-kumar) [inline Frontend_Web_Development_React_Angular_Vue/React/Week 2/2.4_Import,Export,Modules.md:111]: here also same as mentioned above .

**Review by heet-kumar — CHANGES_REQUESTED** [review]
or else if you have link of the image so directly replace here because without image seeing we can't accept your pr .

**Comment by heet-kumar** [comment]
@anandhu720 @Subhampreet @the-pro  review these PR once from your side also if possible.

File: Frontend_Web_Development_React_Angular_Vue/React/Week 2/2.4_Import,Export,Modules.md (hunk 1 above)
Comment (YASHMAROO) [reply Frontend_Web_Development_React_Angular_Vue/React/Week 2/2.4_Import,Export,Modules.md:21]: In my forked repo I can see the images in the preview of the documentation.
<br>
image

File: Frontend_Web_Development_React_Angular_Vue/React/Week 2/2.4_Import,Export,Modules.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+export default function App() {
+  return (
+    <div>
+      <h1>Hello World</h1>
+    </div>
+  );
+}
+```
+
+In the above image, you can see that we have imported **React** from the 'react' path into the file App.js. The second line shows we have imported our stylesheet _style.css_ which is present in the same directory as that of the file displayed in the code.
```
Comment (heet-kumar) [inline Frontend_Web_Development_React_Angular_Vue/React/Week 2/2.4_Import,Export,Modules.md:33]: now its not an image its code so change it .

**Review by heet-kumar — CHANGES_REQUESTED** [review]
do this change also .

**Review by heet-kumar — COMMENTED** [review]
also mention the Reference links and credits to the author from where you've referred this content as per the new rules listed mentioned in server-announcements channel in discord channel of GWOC.

**Review by anandhu720 — APPROVED** [review]
It is fine to me.

**Review by the-pro — APPROVED** [review]

**Pull request merged**